### DISCUSSION TITLE
Hue bulb needs reset after power restore

### DISCUSSION THREAD

**Discussion by M0nk3yOo** [Koenkk/zigbee2mqtt#24083] [Devices]
Hi all,

I hope I didn't miss any old threat, I searched also in the internet.

I'm currently switching some hue bulbs from hue hub, to an already existent zigbee network, build on Home Assistant, z2m and an SONOFF ZBDongle-E 3.0 dongle.

My problem is, that I can in general pair my bulbs (9290012573a) to z2m without any problem. After that I can turn them on, off and switch color. But the issue is, after cuting the power off, and turn it back on one minute later, those bulbs can't rejoin and stay offline.
After that I need to reset the bulb with a Hue dimmer switch. With "permit join" enabled, those bulbs are reconnecting after the bulb reset. Usually the should join automatically, without reset and without keeping the "permit join" enabled.

Does anybody else has this behavior? Any idea to solve that?

Best
M0nk3y

**Comment by blib-blub** [comment]
I have the exact same problem. 
Here - https://github.com/Koenkk/zigbee2mqtt/issues/9920 - the problem seems to be caused by a deconz network running in parallel, which I also have. I will try without the deconz and report back…
